automated install/basic-install.sh

```diff
@@ -413,6 +453,76 @@ package_manager_detect() {
     fi
 }
 
+build_dependency_package(){
+    # This function will build a package that contains all the dependencies needed for Pi-hole
+
+    # remove any leftover build directory that may exist
+    rm -rf /tmp/pihole-meta_*
+
+    # Create a fresh build directory with random name
+    local tempdir
+    tempdir="$(mktemp --directory /tmp/pihole_meta_XXXXX)"
```

Comment:
Shouldn't be an hyphen here, between "pihole" and "meta"?

```suggestion
    tempdir="$(mktemp --directory /tmp/pihole-meta_XXXXX)"
```

Reply:
Good catch.